lib/Conch/DB/Result/HardwareProductProfile.pm

```diff
@@ -94,51 +94,86 @@ __PACKAGE__->table("hardware_product_profile");
   data_type: 'integer'
   is_nullable: 0
 
-=head2 sata_num
+=head2 sata_hdd_num
```

Comment:
The checksums haven't been updated in this file -- did you run 'make dbic', which is the only thing that should be modifying the upper portion of the file?

Reply:
```
daleg@debian9-vm1:~/conch$ git grep "make dbic"
daleg@debian9-vm1:~/conch$
```
ENODOCS on this, so no. I wasn't aware of this.